packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -185,11 +185,6 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWith
    */
   onMenuOpened?: (contextualMenu?: IContextualMenuProps) => void;
 
-  /**
-   * Callback for when the contextualmenu is being closed (removing from the DOM)
-   */
-  onMenuDismissed?: (contextualMenu?: IContextualMenuProps) => void;
-
```

Comment:
Why are we removing this? Other users of the ContextualMenu component may want this functionality

Reply:
Also, this is technically a breaking change (although it's so new I don't think there were any consumers outside of us). This would break our current code that uses onMenuDismissed for ContextualMenu

Reply:
This was added to fix a bug in preview gallery. I have to verify that it's working with the new change. I didn't find other usage for this prop.

Reply:
It's going to be broken since the code is using the prop you are removing